Python/Watermark_Images/watermark_text.py

```diff
@@ -0,0 +1,25 @@
+#Import required Image library
+from PIL import Image, ImageDraw, ImageFont
+
+#Create an Image Object from an Image
+path = '/content/sky.jpg' #main image path
+img = Image.open(path)
+width, height = img.size
+
+draw = ImageDraw.Draw(img)
+text = "©Madhurima.photography" #text on image to add
+
+font = ImageFont.truetype("/content/JMH Typewriter-Thin.ttf", 50)

```

Comment:
We don't have a content directory in here. What are you referring to?

Reply:
The font has to be downloaded and the directory for the same needs to be added. It is optional though.
I have done the changes.